app.js

```diff
@@ -12,8 +13,34 @@ class Dashboard extends React.Component {
     render() {
         return (
             <div>
-                <PsnrResultsChart />
-                <FrameSkipResultsChart />
+                <Chart
+                    graphTitle='PSNR'
+                    graphYAxis='PSNR value'
+                    graphXAxis='Build number'
+                    config={{
+                        resultsUrl: './psnrResults',
+                        resultTransformers: [ psnrTransformer ]
+                    }}
```

Comment:
Do you still prefer having props.config over having props.resultsUrl and props.resultsTransformers?

Reply:
i'd liked this split better because the stuff here in props are final values, while what's in config isn't used as part of the actual chart itself, they're just means to an end...so this division felt cleaner to me.